_data/snippets.yml

```diff
@@ -54,6 +54,10 @@ menu-contribute-overview:
   es:
     title: Contribuciones
     link: /es/contribuciones
+menu-lesson-requests:
+  en:
```

Comment:
@acrymble to add links all pointing to the same page, repeat this yaml block for `es` and `fr`, just copying the same `link` value each time. It would probably be good to translate the `title` values, though.